lxde-base/menu_cache/menu_cache-1.1.0.recipe

```diff
@@ -1,51 +1,55 @@
 SUMMARY="Library used to read freedesktop.org menus"
-DESCRIPTION="Libmenu-cache is a library creating and utilizing caches to\
-speed up the manipulation for freedesktop.org defined application menus.
+DESCRIPTION="Libmenu-cache is a library creating and utilizing caches to \
+speed up the manipulation for freedesktop.org defined application menus. \
 It can be used as a replacement of libgnome-menu of gnome-menus.
 
 Advantages:
 1. Shorten time for loading menu entries.
 2. Ease of use. (API is very similar to that of libgnome-menu)
-3. Lightweight runtime library. (Parsing of the menu definition files 
+3. Lightweight runtime library. (Parsing of the menu definition files
    are done by menu-cache-gen when the menus are really changed.)
 4. Less unnecessary and complicated file monitoring.
 5. Heavily reduced disk I/O."
-HOMEPAGE="https://lxde.org/"
+HOMEPAGE="https://lxde.org"
 COPYRIGHT="LXDE team"
```

Comment:
I think this needs to be a bit more "detailed" :P

Reply:
@alaviss bumped it, not sure it's correct though, not much info ...